utils/updateTitle.js

```diff
@@ -0,0 +1,6 @@
+const { replace, isNil } = require("lodash");
+
+module.exports = (title, prefix = "Copy of") =>
+  replace(title, /(<\w*>)?(.*)/, (_, a, b) => {
+    return isNil(a) ? `${prefix} ${b}` : `${a}${prefix} ${b}`;
+  });
```

Comment:
This would benefit from some unit tests. Be good to test passing a title surrounded by tags, containing tags and without tags so behaviour is documented by example.